doctest failing in 0.2.5.0

```
Running 3 test suites...
Test suite test-doctest: RUNNING...

src/Data/Ini/Config/Raw.hs:167:8: error:
    • Ambiguous type variable ‘f1’ arising from a use of ‘oneOf’
      prevents the constraint ‘(Foldable f1)’ from being solved.
      Probable fix: use a type annotation to specify what ‘f1’ should be.
      These potential instances exist:
        instance Foldable (Either a) -- Defined in ‘Data.Foldable’
        instance Foldable Seq -- Defined in ‘Data.Sequence.Internal’
        instance Foldable Seq.ViewL -- Defined in ‘Data.Sequence.Internal’
        ...plus five others
        ...plus 33 instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In a stmt of a 'do' block: c <- oneOf ";#"
      In the expression:
        do c <- oneOf ";#"
           txt <- T.pack `fmap` manyTill anySingle eol
           return (CommentLine c txt)
      In an equation for ‘sComment’:
          sComment
            = do c <- oneOf ";#"
                 txt <- T.pack `fmap` manyTill anySingle eol
                 return (CommentLine c txt)

src/Data/Ini/Config/Raw.hs:167:14: error:
    • Ambiguous type variable ‘f1’ arising from the literal ‘";#"’
      prevents the constraint ‘(Data.String.IsString
                                  (f1 Char))’ from being solved.
      Probable fix: use a type annotation to specify what ‘f1’ should be.
      These potential instances exist:
        instance (a ~ Char) => Data.String.IsString (Seq a)
          -- Defined in ‘Data.Sequence.Internal’
        instance [safe] (a ~ Tokens s, Data.String.IsString a, Eq a,
                         Stream s, Ord e) =>
                        Data.String.IsString (ParsecT e s m a)
          -- Defined in ‘Text.Megaparsec.Internal’
        instance (a ~ Char) => Data.String.IsString [a]
          -- Defined in ‘Data.String’
        ...plus two instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the first argument of ‘oneOf’, namely ‘";#"’
      In a stmt of a 'do' block: c <- oneOf ";#"
      In the expression:
        do c <- oneOf ";#"
           txt <- T.pack `fmap` manyTill anySingle eol
           return (CommentLine c txt)

src/Data/Ini/Config/Raw.hs:179:31: error:
    • Ambiguous type variable ‘f3’ arising from a use of ‘noneOf’
      prevents the constraint ‘(Foldable f3)’ from being solved.
      Probable fix: use a type annotation to specify what ‘f3’ should be.
      These potential instances exist:
        instance Foldable (Either a) -- Defined in ‘Data.Foldable’
        instance Foldable Seq -- Defined in ‘Data.Sequence.Internal’
        instance Foldable Seq.ViewL -- Defined in ‘Data.Sequence.Internal’
        ...plus five others
        ...plus 33 instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the first argument of ‘some’, namely ‘(noneOf "[]")’
      In the second argument of ‘fmap’, namely ‘some (noneOf "[]")’
      In a stmt of a 'do' block: name <- T.pack `fmap` some (noneOf "[]")

src/Data/Ini/Config/Raw.hs:179:38: error:
    • Ambiguous type variable ‘f3’ arising from the literal ‘"[]"’
      prevents the constraint ‘(Data.String.IsString
                                  (f3 Char))’ from being solved.
      Probable fix: use a type annotation to specify what ‘f3’ should be.
      These potential instances exist:
        instance (a ~ Char) => Data.String.IsString (Seq a)
          -- Defined in ‘Data.Sequence.Internal’
        instance [safe] (a ~ Tokens s, Data.String.IsString a, Eq a,
                         Stream s, Ord e) =>
                        Data.String.IsString (ParsecT e s m a)
          -- Defined in ‘Text.Megaparsec.Internal’
        instance (a ~ Char) => Data.String.IsString [a]
          -- Defined in ‘Data.String’
        ...plus two instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the first argument of ‘noneOf’, namely ‘"[]"’
      In the first argument of ‘some’, namely ‘(noneOf "[]")’
      In the second argument of ‘fmap’, namely ‘some (noneOf "[]")’

src/Data/Ini/Config/Raw.hs:214:30: error:
    • Ambiguous type variable ‘f0’ arising from a use of ‘noneOf’
      prevents the constraint ‘(Foldable f0)’ from being solved.
      Probable fix: use a type annotation to specify what ‘f0’ should be.
      These potential instances exist:
        instance Foldable (Either a) -- Defined in ‘Data.Foldable’
        instance Foldable Seq -- Defined in ‘Data.Sequence.Internal’
        instance Foldable Seq.ViewL -- Defined in ‘Data.Sequence.Internal’
        ...plus five others
        ...plus 33 instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the first argument of ‘some’, namely ‘(noneOf "[]=:")’
      In the second argument of ‘fmap’, namely ‘some (noneOf "[]=:")’
      In a stmt of a 'do' block:
        key <- T.pack `fmap` some (noneOf "[]=:")

src/Data/Ini/Config/Raw.hs:214:37: error:
    • Ambiguous type variable ‘f0’ arising from the literal ‘"[]=:"’
      prevents the constraint ‘(Data.String.IsString
                                  (f0 Char))’ from being solved.
      Probable fix: use a type annotation to specify what ‘f0’ should be.
      These potential instances exist:
        instance (a ~ Char) => Data.String.IsString (Seq a)
          -- Defined in ‘Data.Sequence.Internal’
        instance [safe] (a ~ Tokens s, Data.String.IsString a, Eq a,
                         Stream s, Ord e) =>
                        Data.String.IsString (ParsecT e s m a)
          -- Defined in ‘Text.Megaparsec.Internal’
        instance (a ~ Char) => Data.String.IsString [a]
          -- Defined in ‘Data.String’
        ...plus two instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the first argument of ‘noneOf’, namely ‘"[]=:"’
      In the first argument of ‘some’, namely ‘(noneOf "[]=:")’
      In the second argument of ‘fmap’, namely ‘some (noneOf "[]=:")’

src/Data/Ini/Config/Raw.hs:215:12: error:
    • Ambiguous type variable ‘f2’ arising from a use of ‘oneOf’
      prevents the constraint ‘(Foldable f2)’ from being solved.
      Probable fix: use a type annotation to specify what ‘f2’ should be.
      These potential instances exist:
        instance Foldable (Either a) -- Defined in ‘Data.Foldable’
        instance Foldable Seq -- Defined in ‘Data.Sequence.Internal’
        instance Foldable Seq.ViewL -- Defined in ‘Data.Sequence.Internal’
        ...plus five others
        ...plus 33 instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In a stmt of a 'do' block: delim <- oneOf ":="
      In the expression:
        do pos <- getCurrentLine
           key <- T.pack `fmap` some (noneOf "[]=:")
           delim <- oneOf ":="
           val <- T.pack `fmap` manyTill anySingle eol
           ....
      In an equation for ‘pPair’:
          pPair leading
            = do pos <- getCurrentLine
                 key <- T.pack `fmap` some (noneOf "[]=:")
                 delim <- oneOf ":="
                 ....

src/Data/Ini/Config/Raw.hs:215:18: error:
    • Ambiguous type variable ‘f2’ arising from the literal ‘":="’
      prevents the constraint ‘(Data.String.IsString
                                  (f2 Char))’ from being solved.
      Probable fix: use a type annotation to specify what ‘f2’ should be.
      These potential instances exist:
        instance (a ~ Char) => Data.String.IsString (Seq a)
          -- Defined in ‘Data.Sequence.Internal’
        instance [safe] (a ~ Tokens s, Data.String.IsString a, Eq a,
                         Stream s, Ord e) =>
                        Data.String.IsString (ParsecT e s m a)
          -- Defined in ‘Text.Megaparsec.Internal’
        instance (a ~ Char) => Data.String.IsString [a]
          -- Defined in ‘Data.String’
        ...plus two instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the first argument of ‘oneOf’, namely ‘":="’
      In a stmt of a 'do' block: delim <- oneOf ":="
      In the expression:
        do pos <- getCurrentLine
           key <- T.pack `fmap` some (noneOf "[]=:")
           delim <- oneOf ":="
           val <- T.pack `fmap` manyTill anySingle eol
           ....
src/Data/Ini/Config.hs:504: failure in expression `:{
 data LocalConfig = LocalConfig
   { localUser :: Text }
     deriving (Eq, Show)
:}'
expected:
 but got:
          ^
          <interactive>:35:19: error:
              Not in scope: type constructor or class ‘Text’

Examples: 52  Tried: 2  Errors: 0  Failures: 1
Test suite test-doctest: FAIL
Test suite logged to: dist/test/config-ini-0.2.5.0-test-doctest.log
Test suite test-prewritten: RUNNING...
Passed: basic.ini
Passed: unicode.ini
Test suite test-prewritten: PASS
Test suite logged to: dist/test/config-ini-0.2.5.0-test-prewritten.log
Test suite test-ini-compat: RUNNING...
━━━ Test.Example ━━━
  ✓ propIniEquiv passed 100 tests.
  ✓ propRevIniEquiv passed 100 tests.
  ✓ propIniSelfEquiv passed 100 tests.
  ✓ 3 succeeded.
Test suite test-ini-compat: PASS
Test suite logged to: dist/test/config-ini-0.2.5.0-test-ini-compat.log
2 of 3 test suites (2 of 3 test cases) passed.
```

--- a/src/Data/Ini/Config/Raw.hs
+++ b/src/Data/Ini/Config/Raw.hs
@@ -164,7 +164,7 @@ sBlanks = Seq.fromList <$> many ((BlankLine <$ void eol) <|> sComment)
 
 sComment :: Parser BlankLine
 sComment = do
-  c <- oneOf ";#"
+  c <- oneOf [';', '#']
   txt <- T.pack `fmap` manyTill anySingle eol
   return (CommentLine c txt)
 
@@ -176,7 +176,7 @@ pSection :: Seq BlankLine -> Seq (NormalizedText, IniSection) -> Parser RawIni
 pSection leading prevs = do
   start <- getCurrentLine
   void (char '[')
-  name <- T.pack `fmap` some (noneOf "[]")
+  name <- T.pack `fmap` some (noneOf ['[', ']'])
   void (char ']')
   void eol
   comments <- sBlanks
@@ -211,8 +211,8 @@ pPairs name start leading prevs comments pairs = newPair <|> finishedSection
 pPair :: Seq BlankLine -> Parser (NormalizedText, IniValue)
 pPair leading = do
   pos <- getCurrentLine
-  key <- T.pack `fmap` some (noneOf "[]=:")
-  delim <- oneOf ":="
+  key <- T.pack `fmap` some (noneOf ['[', ']', '=', ':'])
+  delim <- oneOf [':', '=']
   val <- T.pack `fmap` manyTill anySingle eol
   return
     ( normalize key,

